web/styles/reactions.css

```diff
@@ -106,34 +138,27 @@
         }
 
         &:hover {
-            border: 1px solid hsl(200deg 100% 40%);
-            background-color: hsl(195deg 50% 95%);
+            border: 1px solid var(--color-message-reaction-button-border-hover);
+            background-color: var(
+                --color-message-reaction-button-background-hover
+            );
             cursor: pointer;
             opacity: 1;
-            color: hsl(200deg 100% 40%);
+            color: var(--color-message-reaction-button-text-hover);
         }
 
         .message_reaction_count {
-            font-size: 1.1em;
-            color: hsl(0deg 0% 33%);
+            font-weight: 700;
```

Comment:
I think this should stay bold, rather than `var(--message-reaction-font-weight);`? Since the goal is to make our textual `+` have the shape we want. I imagine this will become an icon in the future.

Reply:
Yes, that makes sense to me. We don't want to change what's effectively an icon weight along with anything else